controllers/constants.go

```diff
@@ -1,9 +1,10 @@
 package controllers
 
 const (
-	etcdClientPort = 2379
-	etcdPeerPort   = 2380
-	appName        = "etcd"
-	appLabel       = "app.kubernetes.io/name"
-	clusterLabel   = "etcd.improbable.io/cluster-name"
+	etcdClientPort    = 2379
+	etcdPeerPort      = 2380
+	etcdDataMountPath = "/var/lib/etcd"
```

Comment:
do we need to tell etcd to use this directory via a flag/env var?

Reply:
I think it's the default. Can I deal with this in a follow up branch when I add some e2e tests for this, and know what's required?

Reply:
let's make sure that gets set in the peer pod's env vars